.github/workflows/test.yml

```diff
@@ -16,11 +16,18 @@ jobs:
         ruby:
           - 2.6
           - 2.7
+          - 3.0
```

Comment:
```suggestion
          - "3.0"
```

Reply:
The plus side of this mistake is that you have now already tested with Ruby 3.2. So you might as well add it to the list 😅